### PR TITLE
Api access

### PR DESCRIPTION
**Pull request by gabrielyea**
# Added api access to bookstore project
- Books are loaded from api.
- Books are stored in api.
# Live demo [here](https://gabrielyea.github.io/bookstore)

### CODE REVIEW

File: src/components/booksContainer/BooksContainer.js [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+    dispatch(getBooksFromApi());
+  }, []);
+
+  const createBooks = (booksCollection) => booksCollection.map((book) => {
+    return (
+      <Book
+        key={book.item_id}
+        id={book.item_id}
+        category={book.category}
+        title={book.title}
```
Comment (Davidosky007) [inline src/components/booksContainer/BooksContainer.js:22]: * Kindly fix this error in the console i get them when i try to add new books 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @gabrielyea 👋, 

Good job so far!
* The repo looks professional.✔️
* Git/GitHub flow best practices have been followed✔️
* There are no linter errors✔️
* You have a descriptive PR✔️
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/booksContainer/BooksContainer.js (hunk 1 above)
Comment (gabrielyea) [reply src/components/booksContainer/BooksContainer.js:22]: Done!.. I think... changed the id generator for uuid.

**Review by Emmyn5600 — APPROVED** [review]
Hi, 

Good job so far! 👏🏼 @gabrielyea 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Live link demo works perfectly:heavy_check_mark:
- Connect to API works nice:heavy_check_mark: 

### Optional suggestions
- Validate the title and category input to allow us to add valid information not empty. 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**